api/src/pcapi/routes/backoffice/templates/gdpr_user_extract_data/list_gdpr_user_extract.html

```diff
@@ -0,0 +1,42 @@
+{% from "components/forms.html" import build_filters_form with context %}
+{% import "components/links.html" as links %}
+{% from "components/turbo/lazy_modal.html" import build_lazy_modal with context %}
+{% extends "layouts/connected.html" %}
+{% block page %}
+  <div class="pt-3 px-5 table-container-gdpr_extract">
+    <h2 class="fw-light">Extraction des données jeunes</h2>
+    <div>
+      {% if list_gdpr_data %}
+        <div class="d-flex justify-content-between">
+          <p class="lead num-results">
+            {{ list_gdpr_data | length }}
+            résultat{{ "s" if list_gdpr_data | length > 1 else "" }}
+          </p>
+        </div>
+        <table class="table mb-4"
+               data-table-multi-select-id="table-container-gdpr_extract">
+          <thead>
+            <tr>
+              <th scope="col">ID</th>
+              <th scope="col">Nom et prénom du jeune</th>
+              <th scope="col">Date de création de la demande</th>
+              <th scope="col">État de la demande</th>
+              <th scope="col">Nom et prénom de l’auteur de la demande</th>
+            </tr>
+          </thead>
+          <tbody>
+            {% for gdpr_data in list_gdpr_data %}
+              <tr>
+                <td>{{ gdpr_data.id }}</td>
+                <td>{{ links.build_public_user_name_to_details_link(gdpr_data.user) }}</td>
+                <td>{{ gdpr_data.dateCreated | format_date("%d/%m/%Y") }}</td>
```

Comment:
Pourquoi ne pas garder le formatage par défaut par homogénéité avec l’existant ?

Reply:
j'ai réglé par erreur le code par rapport au test et non le test par rapport au code ( str contre datetime )